go/base/static/js/src/utils.js

```diff
@@ -118,6 +118,17 @@
     return s.charAt(0).toUpperCase() + s.slice(1);
   };
 
+  var non_ascii = function(s) {
```

Comment:
It might be. Whatever we do to the bulk send screen, we should do similarly for dialogues, for the sake of consistency.